packages/analytics/lib/index.js.flow

```diff
@@ -623,7 +623,7 @@ export interface Module extends ReactNativeFirebaseModule {
    * @param name Event name must not conflict with any Reserved Events. 100 characters is the maximum length for param key names.
    * @param params Parameters to be sent and displayed with the event.
    */
-  logEvent(name: string, params?: { [key: string]: string  | number | boolean } | {}): Promise<void>;
+  logEvent(name: string, params?: { [key: string]: any } | {}): Promise<void>;
```

Comment:
The `| {}` part can be removed here I think?

Reply:
Not sure.  It was included also in TypeScript definitions and I don't know Flow that much.  Just noticed that the update was forgotten here as well.

Reply:
My bad, wasn't included in TypeScript.  I removed it.  In any case, the beef of the PR is the code change.